components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -100,20 +101,23 @@ export function WorkspaceEntry({ desc, model }: { desc: WorkspaceInfo, model: Wo
         setChangesModalVisible(true);
     }
     return <div>
-        <a className="rounded-xl whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between hover:bg-gray-100 focus:bg-gitpod-kumquat-light cursor-pointer group" href={startUrl.toString()}>
+        <div className="rounded-xl whitespace-nowrap flex space-x-2 py-6 px-6 w-full justify-between hover:bg-gray-100 focus:bg-gitpod-kumquat-light group">
             <div className="pr-3 self-center">
-                <div className={stateClassName}>
-                    &nbsp;
-            </div>
+                <Tooltip content={state}>
```

Comment:
**issue**: Looks great! The lowercase _stopping_ state is coming from the backend, right? Worth opening a follow up issue to update this to _Stopping_ if possible? WDYT?

Reply:
Yeah, I can do a translation in this PR already.